plugins/woocommerce/includes/admin/settings/views/html-admin-page-shipping-zone-methods.php

```diff
@@ -38,6 +77,17 @@
 						<?php echo wc_help_tip( __( 'These are regions inside this zone. Customers will be matched against these regions.', 'woocommerce' ) ); // @codingStandardsIgnoreLine ?>
 					</label>
 				</th>
+				<td id="wc-shipping-zone-region-picker-root" data-options='<?php echo wp_json_encode( $options ); ?>' data-values='<?php echo wp_json_encode( $locations ); ?>'/>
```

Comment:
Once the above suggested code placement in `WC_Settings_Shipping::zone_methods_screen` is done, `options` and `locations` can be localized to `shippingZoneMethodsLocalizeScript` object, using 
```		
wp_localize_script(
   'wc-shipping-zone-methods',
	'shippingZoneMethodsLocalizeScript',
....
```
 in https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/admin/settings/class-wc-settings-shipping.php#L281-L301